docs/admin/guides/build-image.md

```diff
@@ -7,46 +7,70 @@
 
 Users can add new images to a container repository by uploading a Containerfile. The syntax for
 Containerfile is the same as for a Dockerfile. The same REST API endpoint also accepts a JSON
-string that maps artifacts in Pulp to a filename. Any artifacts passed in are available inside the
-build container at `/pulp_working_directory`.
+string that maps artifacts in Pulp to a filename. Any files passed in (via `build_context`) are
+available inside the build container at the path defined in File Content `relative-path`.
```

Comment:
I don't think this quite captures it. It's more like the files in the build_context repository version are available at there relative path in the build context of the container. Yes this sounds like a tautology (which in turn is a sign that we chose a proper name for it). So we should probably focus more on how the parameter `build_context` defines the actual "build context" the container image is created with.

Note: To make a file actually be present in the running container, you need to first have it in a file repository version, then use the latter as the `build_context` for the image builder feature. Then still, you need to have the `ADD <file> <location in container>` statement.